change/@ni-nimble-components-d350a4f5-967b-4b4d-9f31-96c60b0c63cf.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
In storybook I'm able to tab to the clear button but activating it causes the dropdown to open. Didn't we decide not to allow it to receive tab focus?

And I thought the rationale was that pressing ESC would be the keyboard mechanism to clear the selection, but I'm not seeing that behavior in Storybook either.

(Tested in macOS Firefox)

Reply:
Clear button should no longer be tabbable. Pressing ESC should now clear the `Select` when dropdown is closed (and it is set to `clearable`).